docs/environment.yml

```diff
@@ -19,5 +18,5 @@ dependencies:
 - python-dateutil
 - recommonmark
 - matplotlib
-- bqplot
```

Comment:
I think we need bqplot for the layout page.

Reply:
This deletion is fine; `bqplot` and `numpy` are duplicated.

Reply:
ah well spotted.